src/app/datasets/[id]/DistributionAccordion.tsx

```diff
@@ -0,0 +1,150 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+"use client";
+import React, { useState, useRef, useEffect } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronDown,
+  faChevronUp,
+  faFile,
+} from "@fortawesome/free-solid-svg-icons";
+import { DatasetDistribution } from "@/types/dataset.types";
+
+export default function DistributionAccordion({
```

Comment:
I have seen different ways of typing and exporting React components within this project. Maybe we should align to be consistent